### PR TITLE
Aiven challenges on devportal

### PR DESCRIPTION
**Pull request by maimei**
# What changed, and why it matters

### CODE REVIEW

**Review by carabaestlein — APPROVED** [review]
LGTM!

**Comment by lornajane** [comment]
Please don't merge broken builds to our main branch! I will fix this now, but the build MUST pass to be merged, if you are acting as a maintainer on the project, then please do so responsibly.

**Comment by ftisiot** [comment]
Let me have a look tomorrow morning.

**Pull request merged**
